ovos_core/intent_services/commonqa_service.py

```diff
@@ -1,39 +1,79 @@
-import re
-from threading import Lock, Event
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Framework
```

Comment:
was this accidental? 

this code existed in classic mycroft already and was refactored into a proper service under ovos, this PR is adding support for Sessions which are also a OVOS creation, I don't think this file's license should be changed and attributed to Neon if it's improving existing code. it would be different if we were porting a Neon original creation but this is not new

Reply:
Yeah, missed this when I merged changes from neon-core